python/plugins/processing/tests/testdata/gdal_algorithm_vector_tests.yaml

```diff
@@ -248,3 +248,29 @@ tests:
       OUTPUT:
         name: expected/gdal/dissolved_points.gml
         type: vector
+
+  - algorithm: gdal:convertformat
```

Comment:
It would be good to add a test to https://github.com/qgis/QGIS/blob/master/python/plugins/processing/tests/GdalAlgorithmsVectorTest.py#L67 too -- the command line based tests are MUCH easier to debug vs the ones which actually transform and test layers